web_refresher/static/src/xml/refresher.xml

```diff
@@ -5,13 +5,61 @@
 <template>
     <t t-name="web_refresher.Button">
         <nav class="oe_refresher" aria-label="Refresher" aria-atomic="true">
-            <button
-                class="fa fa-refresh btn btn-icon oe_pager_refresh"
-                aria-label="Refresh"
-                t-on-click="onClickRefresh"
-                title="Refresh"
-                tabindex="-1"
-            />
+            <div class="btn-group" role="group">
+                <div class="btn-group" role="group">
+                    <button
+                        id="auto-refresh-dd"
+                        class="btn btn-secondary dropdown-toggle"
+                        type="button"
+                        data-bs-toggle="dropdown"
+                        aria-expanded="false"
+                    >
+                        Auto Refresh: Off
+                    </button>
+                    <div class="dropdown-menu" aria-labelledby="auto-refresh-dd">
+                        <button
+                            class="dropdown-item"
+                            t-on-click="onChangeAutoRefreshInterval"
+                            value="-1"
+                        >Off</button>
+                        <button
+                            class="dropdown-item"
+                            t-on-click="onChangeAutoRefreshInterval"
+                            value="1000"
+                        >1s</button>
+                        <button
+                            class="dropdown-item"
+                            t-on-click="onChangeAutoRefreshInterval"
+                            value="5000"
+                        >5s</button>
+                        <button
+                            class="dropdown-item"
+                            t-on-click="onChangeAutoRefreshInterval"
+                            value="10000"
+                        >10s</button>
+                        <button
+                            class="dropdown-item"
+                            t-on-click="onChangeAutoRefreshInterval"
+                            value="30000"
+                        >30s</button>
+                        <button
+                            class="dropdown-item"
+                            t-on-click="onChangeAutoRefreshInterval"
+                            value="60000"
+                        >1min</button>
+                    </div>
+                </div>
+                <button
+                    id="manual-refresh-btn"
+                    class="btn btn-secondary m-0"
+                    aria-label="Refresh"
+                    t-on-click="onClickRefresh"
+                    title="Refresh"
+                    tabindex="-1"
+                >
+                    <i id="manual-refresh-icon" class="fa fa-refresh" />
```

Comment:
Moved the icons to an `<i>` to allow the `fa-spin` to be added and removed without spinning the whole button 🤣